src/Sentry/Internal/MainExceptionProcessor.cs

```diff
@@ -49,7 +56,30 @@ public void Process(Exception exception, SentryEvent sentryEvent)
 
                 foreach (var key in sentryException.Data.Keys)
                 {
-                    sentryEvent.SetExtra($"Exception[{i}][{key}]", sentryException.Data[key]);
+                    if (key.StartsWith("sentry", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should probably be `sentry:` instead of `sentry` (i.e. to ignore something like `sentryButNotReally`). Maybe as a constant as well.

Reply:
Not sure about it being a constant since it'll only be used there (unlike the tag/context keys where they are being used by the tests and the Exception extension).